### ISSUE TITLE
Indexer failed when running AzureOpenAIEmbeddingSkill

### ISSUE THREAD

**Issue by LinYe0227** [Azure/azure-search-vector-samples#190]
Hi, I'm following **python Azure AI Search integrated vectorization sample** with exact same configuration. 
The indexer creation step failed with 
**Operation**: "Enrichment.AzureOpenAIEmbeddingSkill.#2" 
**Message** "Could not execute skill because the Web Api request failed"
**Details**:Web Api response status: 'BadRequest', Web Api response details: '{
  "error": {
    "message": "Too many inputs. The max number of inputs is 1.  We hope to increase the number of inputs per request soon. Please contact us through an Azure support request at: https://go.microsoft.com/fwlink/?linkid=2213926 for further questions.",
    "type": "invalid_request_error",
    "param": null,
    "code": null
  }
}
'
Can you please provide any potential solutions? Thanks!

**Comment by LinYe0227** [comment]
Yes, I am using text-embedding-ada-002, which is giving me the error above. I also tried text-embedding-ada-002-v2, got the same error.

**Comment by mattgotteiner** [comment]
Hi can you please share your service name

**Comment by sanket98a** [comment]
Web Api response status: 'BadRequest', Web Api response details: '{
  "error": {
    "message": "Too many inputs. The max number of inputs is 1.  We hope to increase the number of inputs per request soon. Please contact us through an Azure support request at: https://go.microsoft.com/fwlink/?linkid=2213926 for further questions.",
    "type": "invalid_request_error",
    "param": null,
    "code": null
  }
}
'

Same error i am also facing. anyone's prompt attention highly appreciated

**Comment by tahsinac36** [comment]
Running into the same issue as well with ada-002. Do we have any resolution for this now?
